tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionSuite.scala

```diff
@@ -150,4 +150,25 @@ class RegularExpressionSuite extends SparkQueryCompareTestSuite {
     Charset.defaultCharset().name() == "UTF-8"
   }
 
+  private def extractStringsLineEnd(session: SparkSession): DataFrame = {
+    import session.sqlContext.implicits._
+    Seq[(String)](
+      ("123\n"),
```

Comment:
Can we extend this to also include some of the other lineTerminatorChars characters?